2019Labs/CustomSecurityContent/documentation/lab3_profiles.adoc

```diff
@@ -521,36 +525,64 @@ selections:
 .. When you have finished editing,
 press `Ctrl+X`, then enter `y` to save and exit.
 
-. Use templates to generate checks automatically.
+
+=== 3.6.2 Use templates to generate checks automatically
+
+You have successfully defined the rule and added it to the profile.
+However, the rule currently has no check nor remediation.
+That means that Openscap can't check if the Hexchat package is installed.
+You have now two options. You can either write a new check and respective remediations, or you can use an already created template which will generate respective checks and remediations for you.
+The process of writing an OVAL check is described in a different lab, you will use a template in this case.
+
+Templates are a great way of simplifying development of new rules and avoiding unnecessarily large amount of duplicate code.
+There exist sets of rules which are performing very similar checks and can be remediated in similar way.
+For example checking if a certain package is installed, if a certain Systemd service is disabled etc.
+If there were no templates, every rule checking for existence of a package would have to have its own check.
+This would probably result into either code duplication or inconsistency of checks.
```

Comment:
I would rephrase this somehow to make it more relevant to the target audience who just want to write content - currently it has this "what-if" style, I think that rephrasing it to "use templates whenever you can, you will avoid duplication an inconsistencies" may be an improvement.